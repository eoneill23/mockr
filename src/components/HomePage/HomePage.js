import React from 'react';
import './HomePage.css';

export const HomePage = () => {
  return (
    <section className='welcome-info'>
      <p>
        Welcome to Mockr! Mockr is an app that facilitates mock interviews between interviewers and interviewees. Its intuitive design allows an interviewer to ask questions commonly asked in an interview and rate the interviewee's response on a scale from 1-5.
      </p>
      <p>
        It also allows interviewers to email interviewees with their notes from the interview to provide a record of their feedback.
      </p>
      <p>
        Click log in at the top right corner of the page to log in an begin the mock interview process!
      </p>
    </section>
  )
}

export default HomePage;
