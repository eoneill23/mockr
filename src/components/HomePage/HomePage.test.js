import React from 'react';
import { render } from '@testing-library/react';
import HomePage from './HomePage';
import { MockedProvider } from '@apollo/react-testing';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('HomePage', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = render(
      <MockedProvider addTypeName={false}>
        <HomePage />
      </MockedProvider>
    );
  });

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});