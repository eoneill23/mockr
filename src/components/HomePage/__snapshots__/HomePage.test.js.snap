// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should match the snapshot with all data passed in correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="welcome-info"
      >
        <p>
          Welcome to Mockr! Mockr is an app that facilitates mock interviews between interviewers and interviewees. Its intuitive design allows an interviewer to ask questions commonly asked in an interview and rate the interviewee's response on a scale from 1-5.
        </p>
        <p>
          It also allows interviewers to email interviewees with their notes from the interview to provide a record of their feedback.
        </p>
        <p>
          Click log in at the top right corner of the page to log in and begin the mock interview process!
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="welcome-info"
    >
      <p>
        Welcome to Mockr! Mockr is an app that facilitates mock interviews between interviewers and interviewees. Its intuitive design allows an interviewer to ask questions commonly asked in an interview and rate the interviewee's response on a scale from 1-5.
      </p>
      <p>
        It also allows interviewers to email interviewees with their notes from the interview to provide a record of their feedback.
      </p>
      <p>
        Click log in at the top right corner of the page to log in and begin the mock interview process!
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
